### PR TITLE
Passing in the body request option as an array is deprecated in guzzleehttp.  Added form_params for post requests

### PR DESCRIPTION
**Pull request by lowellkitchen**

### CODE REVIEW

File: src/Blue/Tools/Api/Client.php [hunk 1]
```diff
@@ -128,17 +128,19 @@ public function get($apiPath, $queryParams = [])
      * @param $apiPath
      * @param array  $queryParams
      * @param string $data
+     * @param array $formParams
      *
      * @return ResponseInterface
      */
-    public function post($apiPath, $queryParams = [], $data = '')
+    public function post($apiPath, $queryParams = [], $data = '', $formParams = [])
```
Comment (davvblack) [inline src/Blue/Tools/Api/Client.php:135]: When would $data be used here then?

Comment (lowellkitchen) [reply src/Blue/Tools/Api/Client.php:135]: For posting the request body.  The previous version of guzzle allowed $data to be either a string or an array of key/value pairs (for posting form fields).  In the newer version of guzzle, these parameters were separated.

**Review by davvblack — APPROVED** [review]
This change looks good. Is it backwards compatible if someone is still misusing it?

**Comment by lowellkitchen** [comment]
It's not backwards compatible.  It will throw an exception if you try and pass an array in for the $data parameter.  It _could_ easily be made backwards compatible by checking the type of the incoming parameters.

**Pull request merged**